### PR TITLE
Adds my tree

### PR DESCRIPTION
**Pull request by eliocamp**

### CODE REVIEW

**Comment by paocorrales** [comment]
Your name in the file is incorrect.

File: elio.R [hunk 1]
```diff
@@ -1,13 +1,13 @@
 library(flametree)
 
 # your name, without spaces or special characters
-name <- "elio"
+name <- "fern"
```
Comment (paocorrales) [inline elio.R]:
```suggestion
name <- "elio"
```

File: elio.R [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 # pick a seed
-this_seed <- 345357888
+this_seed <- 3465823
 
 # pick some colours
-shades <- c("#d48e15", "#7239b3", "#667885", "#452981")
+shades <- c("#445532", "darkgreen", "orange4", "cyan4")
```
Comment (paocorrales) [inline elio.R:10]: The color palette is terrible!

Comment (eliocamp) [reply elio.R:10]: I like it! Let's leave it that way

File: elio.R (hunk 1 above)
Comment (eliocamp) [reply elio.R]: Thanks!

Comment (eliocamp) [reply elio.R]: I've fixed it.

**Pull request merged**